pajbot/apiwrappers/twitch/helix.py

```diff
@@ -602,3 +602,17 @@ def get_channel_emotes(
             expiry=60 * 60,
             force_fetch=force_fetch,
         )
+
+    def send_chat_announcement(self, bot_id: str, body: Dict[str, str], authorization, channel_id: str = None):
```

Comment:
```suggestion
    def send_chat_announcement(self, sender_id: str, message: str, authorization, channel_id: str):
```
Please also declare a return type